api/database/migrations/exif_invalid_gps_test.go

```diff
@@ -0,0 +1,74 @@
+package migrations_test
+
+import (
+	"os"
+	"math"
+	"bufio"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/photoview/photoview/api/database/migrations"
+	"github.com/photoview/photoview/api/graphql/models"
+	"github.com/photoview/photoview/api/test_utils"
+)
+
+func TestExifMigration(t *testing.T) {
+  envFile, err := os.Open("/home/runner/work/photoview/photoview/api/testing.env")
+	if err != nil {
+	  t.Fatalf("Failed to open environment file: %v", err)
+	}
+	defer envFile.Close()
+
+	scanner := bufio.NewScanner(envFile)
+	for scanner.Scan() {
+		line := scanner.Text()
+		parts := strings.SplitN(line, "=", 2)
+		if len(parts) != 2 {
+		  t.Fatalf("Invalid line in environment file: %s", line)
+		}
+		key, value := parts[0], strings.Trim(parts[1], "'")
+		os.Setenv(key, value)
```

Comment:
Do we not have a function to set env from a file?

Reply:
I don't know) if there is something better, please suggest what would be the replacement)
I'm not a developer and write code based on my general understanding of the functionality and with help from AI)

Reply:
OK. I'll do this later.